client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -33,10 +33,13 @@ import SettingsSidebar from './components/settings-sidebar';
 import { State as OnboardingState } from './stores/onboard/reducer';
 import { STORE_KEY } from './stores/onboard';
 import { Steps } from './types';
-import './stores/domain-suggestions';
-import './stores/verticals-templates';
+import { DomainSuggestions, Verticals, VerticalsTemplates } from '@automattic/data-stores';
 import './style.scss';
 
+DomainSuggestions.register();
+Verticals.register();
+VerticalsTemplates.register();
+
```

Comment:
:heart: 

Wondering if we might be able to isolate some of these even more, e.g. register the `VerticalsTemplates` store from within the Design Selector,

Reply:
Yes, I think so. 8a3f72375686c8403912b33f4f9eaa2154265470 prevents repeated registrations (similar to register on module evaluation) which should help with this. I'll move registration closer to components that use the data.

Reply:
I’m tempted to make the register function return STORE_KEY (and otherwise not export it) so that you have to  register to be able to use the store :thinking_face: 

What I'm seeing right now is this pattern repeated:

```ts
Verticals.register();
const VERTICALS_STORE_KEY = Verticals.STORE_KEY;
```

Reply:
> I’m tempted to make the register function return STORE_KEY (and otherwise not export it)

Implemented in d2c3992ef85b80414bab652f54df95db5f4e29f8

Reply:
Sounds like a great idea!